ocp_utilities/infra.py

```diff
@@ -354,3 +357,65 @@ def create_icsp(icsp_name, repository_digest_mirrors):
     )
     icsp.deploy()
     return icsp
+
+
+def dict_base64_encode(_dict):
+    """Encoding dict in base64
```

Comment:
please add new line after `"""` (For all docstrings)

Reply:
done